extra/drivers/sensor/adis16470/adis16470.c

```diff
@@ -141,41 +130,72 @@ static int adis16470_reg_write(const struct adis16470_data *data, uint8_t reg, u
     return 0;
 }
 
-static int adis16470_reg_read_pair(const struct adis16470_data *data, uint8_t reg[2], uint32_t *value)
+static int adis16470_reg_read_pair(const struct spi_dt_spec *bus, uint8_t reg[2], uint32_t *value)
 {
     uint16_t v[2];
-    int status = adis16470_reg_read(data, reg[0], &v[0]);
+    int status = adis16470_reg_read(bus, reg[0], &v[0]);
     if (status < 0)
         return status;
-    status = adis16470_reg_read(data, reg[1], &v[1]);
+    status = adis16470_reg_read(bus, reg[1], &v[1]);
     if (status < 0)
         return status;
     *value = ((uint32_t)v[1] << 16) | (uint32_t)v[0];
     return 0;
 }
 
-static int adis16470_software_reset(const struct adis16470_data *data)
+static void adis16470_gpio_callback(const struct device *dev, struct gpio_callback *cb, uint32_t pins)
+{
+    struct adis16470_cb_data *cb_data = (struct adis16470_cb_data*)cb;
+    k_sem_give(&cb_data->sem);
```

Comment:
I was confused as to whether -> or & was more dominant in the order of calculation, so it might be better to add parentheses for readability.

計算順序で->と&のどちらが優先度高いか混乱したので、カッコをつける方が可読性は良いかもしれません。

```suggestion
    k_sem_give(&(cb_data->sem));
```

Reply:
対応します！

Reply:
https://github.com/LexxPluss/LexxHard-SensorControlBoard-Firmware/commit/63493394828cf0503c491efdabaa6214110ae996